### PR TITLE
Add a git-cliff config

### PR DESCRIPTION
**Pull request by poljar**

### CODE REVIEW

File: cliff.toml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+body = """
+{% if version %}\
+    ## [{{ version | trim_start_matches(pat="v") }}] - {{ timestamp | date(format="%Y-%m-%d") }}
+{% else %}\
+    ## [unreleased]
+{% endif %}\
+{% for group, commits in commits | group_by(attribute="group") %}
+    ### {{ group | upper_first }}
+    {% for commit in commits %}
+        - {% if commit.breaking %}[**breaking**] {% endif %}{{ commit.message | upper_first }}\
```
Comment (dkasak) [inline cliff.toml:20]:
```suggestion
        - {% if commit.breaking %}[**BREAKING**] {% endif %}{{ commit.message | upper_first }}\
```

File: cliff.toml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    { message = "^fix", group = "Bug Fixes"},
+    { message = "^doc", group = "Documentation"},
+    { message = "^perf", group = "Performance"},
+    { message = "^refactor", group = "Refactor"},
+    { message = "^style", group = "Styling"},
+    { message = "^test", group = "Testing"},
+    { message = "^chore\\(release\\): prepare for", skip = true},
+    { message = "^chore", skip = true},
+    { message = "^ci", skip = true},
+    { body = ".*security", group = "Security"},
```
Comment (dkasak) [inline cliff.toml:52]: Will this catch `Security`? Presumably not if it's an ordinary regex, so in that case

```suggestion
    { body = ".*[sS]ecurity", group = "Security"},
```

**Review by dkasak — APPROVED** [review]
After changes are handled, approved.

File: cliff.toml (hunk 1 above)
Comment (dkasak) [reply cliff.toml:20]: Doesn't hurt to have a bit more prominence?

Comment (poljar) [reply cliff.toml:20]: I don't think I want the changelog to be this shouty.

**Pull request merged**